src/bms/model/bmp.rs

```diff
@@ -1,4 +1,4 @@
-//! This module introduces struct [`BmpObjects`], which manages definitions and events of background image animation (BGI).
+//! This module introduces struct [`BmpObjects`], which manages definitions and events of background image and movie.
```

Comment:
Note that this module doesn't owe to process movie (`video` module is responsible for that). The "animation" word means that there are objects to switch images to animate them.